### PR TITLE
Feature/comment body

### PR DESCRIPTION
**Pull request by vasyldzyuba**

### CODE REVIEW

File: src/app/component/comments/comments.module.ts [hunk 1]
```diff
@@ -4,12 +4,14 @@ import { CommentsComponent } from './components/comments.component';
 import { SharedModule } from '../shared/shared.module';
 import { AddCommentComponent } from '../comments/components/add-comment/add-comment.component';
 import { CommentLikeReplyComponent } from './components/comment-like-reply/comment-like-reply.component';
+import {CommentBodyComponent} from './components/comment-body/comment-body.component';
```
Comment (Gusvarov) [inline src/app/component/comments/comments.module.ts:8]: Please, add spaces between import brackets

File: src/app/component/comments/components/comment-body/comment-body.component.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+  </div>
+  <div class="comment-main-text">
+    <p class="comment-text">In this brilliantly conceived tour de force, Stephen King—who has absorbed the social,
+      political, and popular culture of his generation more imaginatively and thoroughly than
+      any other writer—takes readers on an incredible journey into the past and the possibility
+      of altering it.</p>
+  </div>
+  <app-comment-like-reply></app-comment-like-reply>
+</div>
```
Comment (Gusvarov) [inline src/app/component/comments/components/comment-body/comment-body.component.html:23]: Please, add empty line at the end of file

File: src/app/component/comments/components/comment-body/comment-body.component.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  margin-bottom: -10px;
+}
+
+.like-amount {
+  font-family: "Open Sans";
+  font-size: 12px;
+  line-height: 16px;
+  color: #878787;
+  margin-left: 5px;
+}
```
Comment (Gusvarov) [inline src/app/component/comments/components/comment-body/comment-body.component.scss:77]: same as 23

File: src/app/component/comments/comments.module.ts (hunk 1 above)
Comment (vasyldzyuba) [reply src/app/component/comments/comments.module.ts:9]: done already

Comment (vasyldzyuba) [reply src/app/component/comments/comments.module.ts:9]: done already

File: src/app/component/comments/components/comment-body/comment-body.component.html (hunk 2 above)
Comment (vasyldzyuba) [reply src/app/component/comments/components/comment-body/comment-body.component.html]: done already

File: src/app/component/comments/components/comment-body/comment-body.component.scss (hunk 3 above)
Comment (vasyldzyuba) [reply src/app/component/comments/components/comment-body/comment-body.component.scss]: done already

**Comment by mplesha** [comment]
Reorder imports:
```js
import { Component, OnDestroy, OnInit } from '@angular/core';
import { ActivatedRoute } from '@angular/router';
import { EcoNewsService } from '../../services/eco-news.service';
import { EcoNewsModel } from '../../models/eco-news-model';
import { Subscription } from 'rxjs';
import { singleNewsImages } from '@eco-news-images/single-news-images';
import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
```
rxjs - is third party library.

**Pull request merged**